scripts/mongo/TestTripodQueue.class.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+require_once TRIPOD_DIR.'/mongo/queue/MongoTripodQueue.class.php';
+
+class TestTripodQueue extends MongoTripodQueue {
+
+    protected $mapDatabaseToBaseUri = array(
+        "kent" => "http://resourcelists.kent.ac.uk",
+        "mmu"  => "http://lists.lib.mmu.ac.uk",
+        "ntu"  => "http://resourcelists.ntu.ac.uk",
+        "sta"  => "http://resourcelists.st-andrews.ac.uk",
+        "worc" => "http://readinglists.worcs.ac.uk"
+    );
```

Comment:
Absolutely. I created a 10% pull request to get some visibility of this, but see my comment from above:

```
A 10% pull request - TestTripodQueue is working - but I'm not sure it should even be here.
Perhaps it should be in https://github.com/talis/tripod-php-perf-test-driver
```